.eslintrc.yaml

```diff
@@ -1,26 +1,24 @@
-parser: babel-eslint
+parser: '@typescript-eslint/parser'
 parserOptions:
-  sourceType: module
+  project: './tsconfig.json'
 
 plugins:
+  - '@typescript-eslint'
   - babel
   - react
   - react-native
-  - flowtype
   - react-hooks
 
 extends:
   - eslint:recommended
-  - plugin:flowtype/recommended
   - plugin:react/recommended
+  - 'plugin:@typescript-eslint/recommended'
```

Comment:
```suggestion
  - 'plugin:@typescript-eslint/recommended'
  - 'plugin:@typescript-eslint/recommended-requiring-type-checking'
```

Reply:
@drewvolz I committed that config change and then uncommitted it; we can always add those lints later, but I think I'd like _valid typescript code_ to be the bar we have to cross in order to merge this PR.

Reply:
Sounds good to me!